src/types.ts

```diff
@@ -135,4 +135,21 @@ export interface TableProps {
    * default.
    */
   rowsPerPageOptions?: number[];
+  /**
+   * Whether or not to show the pagination buttons to skip to the first and last
+   * pages. False by default.
+   */
+  showSkipButtons?: boolean;
+}
+
+/**
+ * Big Number Types
+ */
+
+export interface BigNumberProps {
```

Comment:
Lets keep props in the component file. Types is for types that might be/are shared across many components.